### PR TITLE
fix: Override innerText set by OneTrust external script in link

### PR DESCRIPTION
**Pull request by DanielPalafox**

### CODE REVIEW

**Comment by DanielPalafox** [comment]
[Access the Onetrust widget through a link in the site footer](https://trello.com/c/kr6xDzkD/8659-access-the-onetrust-widget-through-a-link-in-the-site-footer)

File: src/app/layout/footer/footer.component.ts [hunk 1]
```diff
@@ -23,7 +23,11 @@ export class FooterComponent implements OnInit {
     })
   }
 
-  ngOnInit() {}
+  ngOnInit() {
+    setTimeout(() => {
+      document.getElementById('ot-sdk-btn').innerText = $localize`:@@footer.cookieSettings:Cookie Settings`
```
Comment (leomendoza123) [inline src/app/layout/footer/footer.component.ts:28]: Dani let's use the window provider to get the document object

Comment (leomendoza123) [reply src/app/layout/footer/footer.component.ts:28]: Also might be a good idea to check if the element exists, before trying to update the `innerText`

Comment (DanielPalafox) [reply src/app/layout/footer/footer.component.ts]: Done! @leomendoza123

**Review by leomendoza123 — APPROVED** [review]

**Pull request merged**